File: docs/src/testers/browser-controls/press_key.ts
// @sniptest filename=press_key.ts
// @sniptest show=1-17
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();

const session = client.Session();
await session.use(async () => {
  await session.execute({ type: "goto", url: "https://example.com" });

  // Press Enter
  await session.execute({ type: "press_key", key: "Enter" });

  // Press Escape
  await session.execute({ type: "press_key", key: "Escape" });

  // Press Tab
  await session.execute({ type: "press_key", key: "Tab" });
});

const status = await session.status();

export { status };

File: docs/src/testers/browser-controls/press_key.py
# @sniptest filename=press_key.py
# @sniptest show=1-15
from notte_sdk import NotteClient

client = NotteClient()

with client.Session() as session:
    session.execute(type="goto", url="https://example.com")

    # Press Enter
    session.execute(type="press_key", key="Enter")

    # Press Escape
    session.execute(type="press_key", key="Escape")

    # Press Tab
    session.execute(type="press_key", key="Tab")

status = session.status()
